### ISSUE TITLE
cli and gui executable name clash on linux

### ISSUE THREAD

**Issue by aogier** [Noovolari/leapp#420]
ciao, I've just installed the cli addon, which is cool so I don't have to clicky clicks things. However, required gui package (for me, that's the deb) install a `/usr/bin/leapp` which then clashes w/ leapp the cli.

To differentiate them using two different command names could probably be the right thing to do.

**Comment by ericvilla** [comment]
Hi @aogier, do you mean you're not able to install the desktop app as /usr/bin/leapp location is already occupied by the CLI? Thanks

**Comment by aogier** [comment]
no, but they clash on my path so I must choose to either rename one of two to eg. leapp-gui or invoke the other w/ npx leapp or something like that

**Comment by aogier** [comment]
also probably yes for the /usr/bin part but I'm not able to confirm that as I don't pollute my system with `npm install -g` stuff
